src/ng1/services/colorService/colorService.service.js

```diff
@@ -1,51 +1,39 @@
 export default function $colorService() {
 
 	var $colorService = {};
+	var colorSet = colorSets.keppel;
+	var html;
+	var element;
+	var colors;
 
-	var html = '<div class="primary-color"></div>'+
-				'<div class="accent-color"></div>'+
-				'<div class="secondary-color"></div>'+
-				'<div class="alternate1-color"></div>'+
-				'<div class="alternate2-color"></div>'+
-				'<div class="alternate3-color"></div>'+
-				'<div class="vibrant1-color"></div>'+
-				'<div class="vibrant2-color"></div>'+
-				'<div class="grey1-color"></div>'+
-				'<div class="grey2-color"></div>'+
-				'<div class="grey3-color"></div>'+
-				'<div class="grey4-color"></div>'+
-				'<div class="grey5-color"></div>'+
-				'<div class="grey6-color"></div>'+
-				'<div class="grey7-color"></div>'+
-				'<div class="grey8-color"></div>'+
-				'<div class="chart1-color"></div>'+
-				'<div class="chart2-color"></div>'+
-				'<div class="chart3-color"></div>'+
-				'<div class="chart4-color"></div>'+
-				'<div class="chart5-color"></div>'+
-				'<div class="chart6-color"></div>'+
-				'<div class="ok-color"></div>'+
-				'<div class="warning-color"></div>'+
-				'<div class="critical-color"></div>'+
-				'<div class="partition1-color"></div>'+
-				'<div class="partition9-color"></div>'+
-				'<div class="partition10-color"></div>'+
-				'<div class="partition11-color"></div>'+
-				'<div class="partition12-color"></div>'+
-				'<div class="partition13-color"></div>'+
-				'<div class="partition14-color"></div>'+
-				'<div class="social-chart-node-color"></div>'+
-				'<div class="social-chart-edge-color"></div>';
-
-	var element = document.createElement('div');
-	element.className = 'color-chart';
-	element.innerHTML = html;
-
-	document.body.appendChild(element);
+	function setColors() {
+
+		html = '';
+
+		for (let key in colorSet) {
+            html += '<div class="' + colorSet[key] + '-color"></div>';
+        }
```

Comment:
Indentation is still a bit wonky here, as you can see on GitHub. This is because the file incorrectly uses tabs instead of spaces.